Applications/Utils/MeshEdit/removeMeshElements.cpp

```diff
@@ -67,9 +76,19 @@ int main (int argc, char* argv[])
 	TCLAP::MultiArg<std::string> eleTypeArg("t", "element-type",
 	                                      "element type to be removed", false, "element type");
 	cmd.add(eleTypeArg);
-	TCLAP::MultiArg<unsigned> matIDArg("m", "material-id",
-	                                      "material id", false, "material id");
-	cmd.add(matIDArg);
+
+	TCLAP::MultiArg<char> char_property_arg(
```

Comment:
Character properties can be written from the tool `ResetPropertyInPolygonalRegion`. For this reason I added these property type here.